bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
I guess we won't be renaming the file to `opam`, so we can remove the `name:` fields.